src/codegen/utilities/interpreter_helpers.py

```diff
@@ -366,12 +365,15 @@ def create_compound_parameter_request(func):
     return f"grpc_types.{compound_parameter_type}(" + ", ".join(parameters) + ")"
 
 
-def get_response_parameters(output_parameters: list):
+def get_response_parameters(output_parameters: list, is_read_method):
```

Comment:
As mentioned in other PRs, passing `func` into the function would allow you to change its behavior without changing its signature.